test/legacy_test/test_imperative_optimizer_v2.py

```diff
@@ -790,7 +791,8 @@ def get_optimizer(self):
         return optimizer
 
     # should fix: may fail in CI-windows
-    def _test_lamb(self):
+    @test_with_pir_api
+    def test_lamb(self):
```

Comment:
```suggestion
    def _test_lamb(self):
```

这行 CI-windows 会出问题，还是复原吧，此 pr 不做修复